package/skelcd-control-SLES4SAP.spec

```diff
@@ -65,10 +70,8 @@ SLES4SAP control file needed for installation
 
 %build
 # transform ("patch") the original SLES installation file
-xsltproc %{SOURCE0} /installation.xml > installation.xml
-#xsltproc %{SOURCE0} /installation.xml > installation1.xml
-#xmllint --format installation1.xml > installation.xml
-diff -u /installation.xml installation.xml || :
+xsltproc %{SOURCE0} %{skelcd_control_datadir}/SLES.xml > installation.xml
+diff -u %{skelcd_control_datadir}/SLES.xml installation.xml || :
```

Comment:
@lslezak please, take a look to this two lines. Looks good for you?

I could build the package successfully.